model/flow/epoch.go

```diff
@@ -289,22 +292,22 @@ func NewEpochStatus(firstBlockID, currentSetup, currentCommit, nextSetup, nextCo
 		},
 	}
 
-	err := status.check()
+	err := status.Check()
```

Comment:
It was always possible for an `EpochStatus` to become invalid after construction if it is misused (it is a mutable data structure after all). 

The main reason for this change was to avoid creating 6 temporary local variables representing each event ID during construction of the `EpochStatus` in `bootstrapEpoch`. Instead we add each event ID to the `EpochStatus` as we go, then call `Check` afterward as a sanity check.